longevity_test.py

```diff
@@ -325,7 +325,7 @@ def _pre_create_schema(self, keyspace_num=1, in_memory=False):
             self.log.debug('{} Created'.format(keyspace_name))
             self.create_cf(session,  'standard1', key_type='blob', read_repair=0.0, compact_storage=True,
                            columns={'"C0"': 'blob', '"C1"': 'blob', '"C2"': 'blob', '"C3"': 'blob', '"C4"': 'blob'},
-                           in_memory=in_memory)
+                           in_memory=in_memory, scylla_encryption_options=scylla_encryption_options)
```

Comment:
@amoskong I don't know if it's the right place to use it, or at least the only place to use it.
In case someone would want to use if for other schemas than default c-s, it wouldn't work.

Reply:
It's not added a general interface to setup encryption, just for satisfy the request of one encryption at-rest testing.

Do you expected to configure or enable encryption at-rest in yaml configure, and all use tables will be effected?